{
  try {
    execute();
  }
 catch (  IOException ex) {
    LogLog.warn("Exception during file rollover.",ex);
  }
}
